darwin/cli_functions.py

```diff
@@ -237,6 +269,45 @@ def pull_dataset(dataset_slug: str):
     print(f"Dataset {release.identifier} downloaded at {dataset.local_path}. ")
 
 
+def migrate_dataset(dataset_slug: str):
+    """Migrates an outdated local dataset to the latest format.
+
+    Parameters
+    ----------
+    dataset_slug: str
+        Slug of the dataset to which we perform the operation on
+    """
+    identifier = DatasetIdentifier.parse(dataset_slug)
+    client = _load_client(offline=True)
+    for p in client.list_deprecated_local_datasets(identifier.team_slug):
```

Comment:
while this check is great, what about just checking if the dataset is already migrated first, so we can say "already migrated".